### DISCUSSION TITLE
大佬自己挂载的OneDrive是什么版本的哈?我们公司似乎屏蔽了onedrive,但是大佬的的可以哈~~

### DISCUSSION THREAD

**Discussion by jsjcjsjc** [spencerwooo/onedrive-vercel-index#851] [Q&A] [answered]
想问一下,我也去申请一个~~

**Comment by spencerwooo** [comment, marked as answer]
一般用这个项目的应该都是要么普通的国际版、教育版或者国内的世纪互联版 OneDrive，要么是申请的 Microsoft 365 E5 开发者账号 [^1] 那种。演示站用的是后者，E5 开发者版。

[^1]: https://developer.microsoft.com/en-us/microsoft-365/dev-program
